community/neo4j/src/test/java/upgrade/StoreUpgradeOnStartupTest.java

```diff
@@ -78,10 +78,10 @@
     public static Collection<Object[]> versions()
     {
         return Arrays.asList(
-                new Object[]{Legacy20Store.LEGACY_VERSION},
-                new Object[]{Legacy21Store.LEGACY_VERSION},
-                new Object[]{Legacy22Store.LEGACY_VERSION},
-                new Object[]{Legacy23Store.LEGACY_VERSION}
+                new Object[]{LowLimitV2_0.STORE_VERSION},
```

Comment:
Each parameter can be plain `String`, there is no need to wrap it into `Object[]`